CONTRIBUTING.md

```diff
@@ -336,6 +336,30 @@ $ make test
 
 Don't forget to update the `CHANGELOG.md`.
 
+###### How to update terraform-provider-azurerm
+
+> Only if azurerm provider need to updated
+
+Since terraform-provider-azurerm moved, source code under `internal`, Terracognita use a fork including a commit to create [azurerm/provider.go](https://github.com/cycloidio/terraform-provider-azurerm/commit/56a7990887012bc71e3080b176d6c10afc27a8cc) file.
+The process to update terraform-provider-azurerm is the following
+
+```
+git clone git@github.com:cycloidio/terraform-provider-azurerm.git
+cd terraform-provider-azurerm
+git remote add upstream https://github.com/hashicorp/terraform-provider-azurerm.git
+git fetch upstream
+git checkout main
+git rebase upstream/main
+git push -f origin main
```

Comment:
I would replace `main` for something else hehe.

Reply:
updated to cycloid